frontend/views/containers/proposals/ProposalsWidget.vue

```diff
@@ -117,11 +117,8 @@ export default ({
     hadVoted (proposal) {
       return proposal.votes[this.currentIdentityState.attributes.username] || proposal.status !== STATUS_OPEN
     },
-    openModal (modal) {
-      sbp('okTurtles.events/emit', OPEN_MODAL, modal)
-    },
-    seeAll () {
-      // Todo
+    openModal (modal, queries) {
+      sbp('okTurtles.events/emit', OPEN_MODAL, modal, queries)
```

Comment:
Instead of passing in the proposals to `PropositionsAllModal` via the URL `query` string, could you simply retrieve them from the state the same way `ProposalsWidget` does?